Rx/v2/src/rxcpp/operators/rx-repeat.hpp

```diff
@@ -25,166 +25,46 @@
 #define RXCPP_OPERATORS_RX_REPEAT_HPP
 
 #include "../rx-includes.hpp"
+#include "impl-retry-repeat.hpp"
```

Comment:
rxcpp was structured so that all include dependencies are expressed in `rx-includes.hpp` and a few of the `.hpp` it directly includes (sources, operators, etc..). RX-LITE changed the include structure to make the individual operator includes optional and move them later in the dependency graph.

some options are:

- leave this include structure as is. 
 - new pattern
- move this include to `rx-includes.hpp` 
 - will always be included even in RX-LITE when retry and repeat are not
- merge repeat and retry into a single `.hpp`
 - bleeds the implementation details to the user

Without RX-LITE I would choose differently, but I am leaning to keeping this include as is. What do you think?